dataland-frontend/tests/e2e/specs/low/pages/LandingPage.ts

```diff
@@ -65,4 +59,38 @@ describe('Footer Section', () => {
             .url().should('include', '/dataprivacy')
         cy.get("h2").contains('Data Privacy')
     })
-})
\ No newline at end of file
+})
+
+describe('Sample Section', () => {
+    it('Check that the sample section works properly with authentication', () => {
```

Comment:
Die beiden Tests sind doch identisch bis auf einmal mit und einmal ohne login. Das heißt der code in den beiden Tests ist dupliziert.

Reply:
wurde geändert